### PR TITLE
JET-147: hardcode minimum for partial liquidations

### PR DESCRIPTION
**Pull request by suscd**
Add a minimum value, under which there's no constraint that a partial
amount of the collateral is liquidated. This is useful to prevent cases
where accounts are slowly liquidated to a point where the remaining
balances are too small and get rounded to zero when attempting to repay
via liquidation.

### CODE REVIEW

File: programs/jet/src/state/obligation.rs [hunk 1]
```diff
@@ -35,6 +35,11 @@ use super::MarketReserves;
 /// Limit the total positions that can be registered on an obligation
 const MAX_OBLIGATION_POSITIONS: usize = 11;
 
+/// The minimum quote value for an obligation to require partial liquidations,
+/// if an obligation has a lower value then it can be fully liquidated when below
+/// the minimum collateralization ratio.
+const MIN_PARTIAL_LIQUIDATION_VALUE: u64 = 10;
```
Comment (nevi-me) [inline programs/jet/src/state/obligation.rs:41]: @suscd is this 0.10? I'm still getting the hang of how the `Number` conversions work

Comment (eyevz) [reply programs/jet/src/state/obligation.rs:41]: This appears to be an amount in the numeraire currency, USD.

**Review by eyevz — APPROVED** [review]
Looks okay to me.

**Comment by KevinRafaelConnor** [comment]
Hey guys, just a reminder to tag the Jira issue (in this case, [JET-147](https://jetprotocol.atlassian.net/browse/JET-147?atlOrigin=eyJpIjoiNThkZGY1NGI0MGEzNGMxZjlhODFmMWI5Y2Q5NTVlMWUiLCJwIjoiaiJ9)) when doing commits, merges, and pull requests

**Review by nevi-me — APPROVED** [review]
LGTM

**Pull request merged**